docs/REST API/rest-api.md

```diff
@@ -3931,7 +3931,7 @@ This endpoint tells Fleet to run a custom an MDM command, on the targeted macOS
 
 | Name                      | Type   | In    | Description                                                               |
 | ------------------------- | ------ | ----- | ------------------------------------------------------------------------- |
-| command                   | string | json  | A base64-encoded MDM command as described in [Apple's documentation](https://developer.apple.com/documentation/devicemanagement/commands_and_queries) |
+| command                   | string | json  | A base64-encoded MDM command as described in [Apple's documentation](https://developer.apple.com/documentation/devicemanagement/commands_and_queries). Supported formats are standard (RFC 4648) and raw (unpadded) encoding (RFC 4648 section 3.2) |
```

Comment:
Might be helpful to add a link to the specification this is referencing— maybe [Wikipedia](https://en.wikipedia.org/wiki/Base32#RFC_4648_Base32_alphabet), or [this](https://www.rfc-editor.org/info/rfc4648) if it's the official source? (Won't let this be a blocker for merging, just a nice-to-have.)

Reply:
that sounds like a great idea. I will merge as-is so I don't lose the approval from George and follow up shortly with a PR.